src/components/Reviews.jsx

```diff
@@ -1,10 +1,13 @@
 import {useState} from 'react'
 import ReviewCard from './ReviewCard'
+import SingleReview from './SingleReview'
 
 const Reviews = () => {
     const [reviews, setReviews] = useState([])
     return (
+        <main>
         <ReviewCard reviews={reviews} setReviews={setReviews}/>
+        </main>
     )
```

Comment:
This isn't connected to this ticket but I would assume the component `Reviews` would contain a list of the `ReviewCard` components, rather than a single `ReviewCard` component that contains all the reviews
I.e. this isn't mandatory but it would make more sense to have your map in your Reviews component